### PR TITLE
Advanced WSGI logging

### PR DESCRIPTION
**Pull request by zopyx**
Work in progress

### CODE REVIEW

**Comment by zopyx** [comment]
@jenkins-plone-org please run jobs

**Comment by zopyx** [comment]
@jenkins-plone-org please run jobs

**Comment by tschorr** [comment]
@jenkins-plone-org please run jobs

File: news/119.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - access-log-handler 
+  - access-log-args 
+  - access-log-kwargs 
+  - event-log-handler 
+  - event-log-args
+  - event-log-kwargs
+  [ajung]
+
+- added documentation and examples for advanced WSGI logger configuration
+  [ajung]
```
Comment (jensens) [inline news/119.feature]: in a news file, all must be in one line (yeah, annoying, I know). To split in multiple entries just create multiple files, like 119-1.feature, 119-2.doc, ...

Comment (tschorr) [reply news/119.feature]: @zopyx could you sort this out?

**Comment by zopyx** [comment]
Thanks @tschorr

**Pull request merged**